### ISSUE TITLE
Make benchmarks of Sailor

### ISSUE THREAD

**Issue by Etiene** [Etiene/sailor#55]
Sailor needs some benchmarks to compare with other tools!
*contributors wanted for this*

**Comment by pkisztelinski** [comment]
If you want to compare sailor with other tools than you should look at this -> https://github.com/TechEmpower/FrameworkBenchmarks  
https://www.techempower.com/benchmarks/

**Comment by Etiene** [comment]
Yes! This is a good tool :)  It would be great to have someone contributing with this! :)
